app/controllers/api/account.php

```diff
@@ -1716,7 +1716,7 @@
         }
 
         $url = Template::parseURL($url);
-        $url['query'] = Template::mergeQuery(((isset($url['query'])) ? $url['query'] : ''), ['userId' => $user->getId(), 'secret' => $verificationSecret, 'expires' => $expireTime]);
+        $url['query'] = Template::mergeQuery(((isset($url['query'])) ? $url['query'] : ''), ['userId' => $user->getId(), 'secret' => $verificationSecret, 'expires' => $expires]);
```

Comment:
Last minute catch, we use the `expire` key everywhere, but that URL passes `expires` 🤔